code/modules/martial_arts/krav_maga.dm

```diff
@@ -161,17 +166,35 @@
 	if(H.get_item_by_slot(SLOT_HUD_GLOVES) == src)
 		style.remove(H)
 
-/obj/item/clothing/gloves/color/black/krav_maga/sec//more obviously named, given to sec
+//////////////////////////////
+// Warden gloves
+//////////////////////////////
+/obj/item/clothing/gloves/color/black/krav_maga/sec
 	name = "Krav Maga gloves"
-	desc = "These gloves can teach you to perform Krav Maga using nanochips."
+	desc = "These gloves can teach you to perform Krav Maga using nanochips for as long as you're wearing them."
 	icon_state = "fightgloves"
 	item_state = "fightgloves"
 
 /obj/item/clothing/gloves/color/black/krav_maga/sec/Initialize(mapload)
 	. = ..()
 	RegisterSignal(src, COMSIG_PARENT_QDELETING, PROC_REF(alert_admins_on_destroy))
 
-/// for nukies
+/obj/item/clothing/gloves/color/black/krav_maga/sec/examine_more(mob/user)
+	..()
+	. = list()
```

Comment:
oh that's clever. I quite like this as a way to reset things.

Reply:
Learned this when working on the energy weapons PR. Avoided the requirement to store the extended descriptions as a var on the item itself whilst still preventing children items from displaying parent descriptions.